documentation/general/dash-sonic-hld.md

```diff
@@ -283,7 +283,19 @@ pl_sip_encoding          = Private Link encoding for IPv6 SIP transpositions; Fo
 pl_underlay_sip          = Underlay SIP (ST GW VIP) to be used for all private link transformation for this ENI
 ```
 ### 3.2.4 ACL
-  
+
+```
+DASH_PREFIX_TAG_TABLE:{{tag_name}}
```

Comment:
Can we add some scale numbers for the ACL tags in the HLD ? 
 - How many total tags ?
 - How many prefixes per tag ?
 - How many tags can each prefix belong to ?

Reply:
We will need to update the HLD with the scale numbers.  
Possibly by sometime this week 3/16/2023
@prsunny

Reply:
Added the scale numbers of tags in the above scale table.